core/memory/arena/cc/arena.cpp

```diff
@@ -143,7 +143,6 @@ void* Arena::allocate(uint32_t size, uint32_t align) {
 
     // posix_memalign states that the alignment must be a multiple
     // of sizeof(void*).
-    align = std::max(align, static_cast<uint32_t>(sizeof(void*)));
     align = std::max(align, page_size_);
```

Comment:
You mean to remove the alignment statement here? You still have the comment. If you removed it because `sizeof(void*) < page_size_`, how about a `static_assert`?